sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/roberta-base/scripts/requirements.txt

```diff
@@ -0,0 +1,7 @@
+transformers == 4.21.1
+accelerate
+torch >= 1.3
+datasets >= 1.8.0
+sentencepiece != 0.1.92
+protobuf
```

Comment:
Are you running into any issues with protobuf not being available in the pytorch containers ?

Reply:
No I didn't run into issue and I copy this requirements.txt from [here](https://github.com/huggingface/transformers/blob/main/examples/pytorch/language-modeling/requirements.txt).
Should I delete protobuf

Reply:
We only need `transformers` and `datasets`, delete the rest